packages/editor/src/client/routes/commentList.tsx

```diff
@@ -0,0 +1,408 @@
+import React, {useEffect, useState} from 'react'
+
+import {
+  CommentRefFragment,
+  useCommentListQuery,
+  useApproveCommentMutation,
+  useRequestChangesOnCommentMutation,
+  CommentState,
+  CommentRejectionReason,
+  Comment,
+  useRejectCommentMutation,
+  CommentSort
+} from '../api'
+import {
+  Timeline,
+  FlexboxGrid,
+  Input,
+  InputGroup,
+  Icon,
+  IconButton,
+  Table,
+  Modal,
+  Button,
+  Dropdown,
+  Alert
+} from 'rsuite'
+
+import {DescriptionList, DescriptionListItem} from '../atoms/descriptionList'
+import {RichTextBlock} from '../blocks/richTextBlock/richTextBlock'
+
+import {useTranslation} from 'react-i18next'
+
+import {DEFAULT_TABLE_PAGE_SIZES, mapTableSortTypeToGraphQLSortOrder} from '../utility'
+
+const {Column, HeaderCell, Cell, Pagination} = Table
+
+enum ConfirmAction {
+  Approve = 'approve',
+  RequestChanges = 'requestChanges',
+  Reject = 'reject'
+}
+
+function mapCommentActionToColor(currentAction?: ConfirmAction) {
+  switch (currentAction) {
+    case ConfirmAction.Approve:
+      return 'green'
+    case ConfirmAction.RequestChanges:
+      return 'yellow'
+    case ConfirmAction.Reject:
+      return 'red'
+    default:
+      return 'green'
```

Comment:
bad practice. I'd return default color or none color if the type is unknown. Or make the parameter `currentAction` not optional i think then you don't need a default case.